tardis/transport/montecarlo/base.py

```diff
@@ -167,6 +167,11 @@ def run(
         self.transport_state = transport_state
 
         number_of_vpackets = self.montecarlo_configuration.NUMBER_OF_VPACKETS
+        ENABLE_RPACKET_TRACKING = (
+            True
+            if self.montecarlo_configuration.ENABLE_RPACKET_TRACKING
+            else None
+        )
```

Comment:
Could this setup for booleans be factored out into the montecarlo configuration class?

Reply:
I tried that but it seems to be not working for jitclass. I will try again tomorrow and will let you know the exact error.

Reply:
Do the changes I made were the one you were suggesting?